src/components/HomeUpload/HomeUpload.js

```diff
@@ -11,8 +11,33 @@ import AppContext from "../../AppContext";
 export default function HomeUpload() {
   const [files, setFiles] = useState([]);
   const { apiUrl } = useContext(AppContext);
+  const [directoryMode, setDirectoryMode] = useState(false);
+
+  useEffect(() => {
+    if (directoryMode) {
+      inputRef.current.setAttribute("webkitdirectory", "true");
```

Comment:
Don't we also need `multiple` and ``mozdirectory`?

Reply:
We don't need `multiple` because it's already set. 
We don't need `mozdirectory` because mozilla supports `webkitdirectory` https://caniuse.com/#feat=mdn-api_htmlinputelement_webkitdirectory